code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
You forgot about the part where said person is also carrying around a bottle of lube to slip the incoming officers and added an easy-to-make silencing chem so he could never call for backup in the first place.

Analyzing a situation, when it comes to chems, purely around one powergamey chem, when there's a plethora of them is missing the bigger picture, at best, and dishonest, at worst.